content/en/docs/distributions/gke/deploy/upgrade.md

```diff
@@ -256,9 +256,18 @@ To upgrade from specific versions of Kubeflow, you may need to take certain manu
 Kubeflow on Google Cloud doesn't guarantee the upgrade for each Kubeflow component always works with the general upgrade guide here. Please refer to corresponding repository in [Kubeflow org](https://github.com/kubeflow) for upgrade support.
 {{% /alert %}}
 
+### Upgrade Kubeflow cluster to v1.6
+
+Starting from Kubeflow v1.6.0:
+* Component with deprecated API versions were upgraded to support GKE v1.22. If you would like to upgrade your GKE cluster, follow [GCP instructions](https://cloud.google.com/kubernetes-engine/docs/how-to/upgrading-a-cluster).
+* ASM was upgraded to v1.14. Follow the instructions on how to [upgrade ASM (Anthos Service Mesh)](#upgrade-asm-anthos-service-mesh). If you want to use ASM version prior to 1.11, refer to [the legacy instructions](https://github.com/kubeflow/gcp-blueprints/blob/master/kubeflow/common/asm/deprecated/README.md). 
+* Knative was upgraded to v1.2. Follow [Knative instructions](https://knative.dev/docs/install/upgrade/upgrade-installation/) to check current version and see if the update includes any breaking changes.
+* Cert-manager was upgraded to v1.5. To check your current version and see if the update includes any breaking changes, follow the [cert-manager instructions](https://cert-manager.io/docs/installation/upgrading/). 
+* Deprecated kfserving component was removed. To upgrade to KServe, follow the [KServe Migration guide](https://github.com/kserve/kserve/tree/master/hack/kserve_migration).
+
 ### Upgrade Kubeflow cluster to v1.5
 
-Starting from Kubeflow v1.5.1 we upgraded ASM to v1.13. Follow the instructions on how to [upgrade ASM (Anthos Service Mesh)](#upgrade-asm-anthos-service-mesh). If you want to use ASM version prior to 1.11, refer to [the legacy instructions](https://github.com/kubeflow/gcp-blueprints/blob/master/kubeflow/common/asm/deprecated/README.md).
+In Kubeflow v1.5.1 we use ASM to v1.13. See [how to upgrade ASM](#upgrade-asm-anthos-service-mesh). To use ASM versions prior to 1.11, follow [the legacy instructions](https://github.com/kubeflow/gcp-blueprints/blob/master/kubeflow/common/asm/deprecated/README.md).
```

Comment:
Change `we use ASM to v1.13` to `we use ASM v1.13`

Reply:
Fixed